src/fsharp/FSharp.Core/string.fsi

```diff
@@ -18,6 +18,29 @@ namespace Microsoft.FSharp.Core
     [<RequireQualifiedAccess>]
     module String = 
 
+        /// <summary>Builds a new string whose characters are the results of applying the function <c>mapping</c>
+        /// to each of the characters of the input string and concatenating the resulting
+        /// strings.</summary>
+        ///
+        /// <param name="mapping">The function to produce a string from each character of the input string.</param>
+        /// <param name="str">The input string.</param>
+        ///
+        /// <returns>The concatenated string.</returns>
+        /// 
+        /// <example> The following samples shows how to interspace spaces in a text
```

Comment:
This is actually the first time I've seen multiple examples in the wild, so congratulations on surprising me :)

I did check, and FSharp.Formatting and FsAutoComplete _do_ handle multiple examples just fine, so this shouldn't be a problem from the perspective of tooling consuming these docs.

@dsyme it does make me want some kind of 'example identifier' for use in generation, for example generating a link of `FSharp.Core.StringModule#collect-example-collect-space` in order to jump to the specific example, but that's just some off-the-cuff musing.

Reply:
> @dsyme it does make me want some kind of 'example identifier' for use in generation, for example generating a link of FSharp.Core.StringModule#collect-example-collect-space in order to jump to the specific example, but that's just some off-the-cuff musing. (FSharp.Formatting already generates a numeric identifier for this case, but something more stable might be useful?)

Makes sense, yes, even just `collect-example-1`

Reply:
just use `<example id="collect-example-whatever">` I think

Reply:
> I did check, and FSharp.Formatting and FsAutoComplete do handle multiple examples just fine, so this shouldn't be a problem from the perspective of tooling consuming these docs.

Thank you for checking this!!!!

Reply:
yeah, that's what I was getting at, I just wonder if it's worth codifying that in the tooling RFC and adding support to FSharp.Formatting then.

Reply:
> yeah, that's what I was getting at, I just wonder if it's worth codifying that in the tooling RFC and adding support to FSharp.Formatting then.

Absolutely

Reply:
tracking this [on the RFC](https://github.com/fsharp/fslang-design/pull/630) and in [FSharp.Formatting](https://github.com/fsprojects/FSharp.Formatting/pull/704)

Reply:
Added ids to every example tag

Reply:
Facepalm you want to generate the ids ...

Reply:
no, I'd keep them. My intent was for the author to give them meaningful ids. There is some last-chance id generation happening in FSharp.Formatting, but the MR I have up now always takes the `id` that is present if there is one.  So, all in all, I think what you have now is great 👍